client/js/components/user/CustomerSettings/CustomerSettings.vue

```diff
@@ -183,6 +183,12 @@
         <customer-settings-support />
       </customer-settings-section>
 
+      <customer-settings-section
+        v-if="hasPermission('feature_customer_login_support_contact_administration')"
+        :title="Translator.trans('support.technical')">
```

Comment:
why we use here technical support as name? its irritating because in introduction page technical support is a static support, who you cannot create as mandant admin. I would suggest here something else. which members will be added here by the mandant admin as contact?

Reply:
have you changed here anything because of the name?

Reply:
Thanks for the answer. I will think about and talk to a po and than inform you

Reply:
discussed here https://demosdeutschland.slack.com/archives/C05BV588CS0/p1698694605825529
changed here 27780dbe53106125c422a6c9490c2a20c3364c98